Triangulation_3/include/CGAL/Triangulation_segment_traverser_3.h

```diff
@@ -0,0 +1,1110 @@
+// Copyright (C) 2012  Utrecht University
```

Comment:
This is an issue that needs to be sorted out. @afabri I remember some email exchange mentioning that Utrecht was OK to give the change to INRIA, can you dig in your emails?

Reply:
For the record, we simply need to add the Utrecht the copyright notice.